app/assets/javascripts/spina/controllers/trix_controller.js

```diff
@@ -33,9 +33,12 @@ export default class extends Controller {
   }
   
   insertAttachment(event) {
-    let attachment = new Trix.Attachment({content: `<span class="trix-attachment-spina-image" data-label="Alt text">
-      <img src="${event.detail.embeddedUrl}" />
-    </span>`, contentType: "Spina::Image"})
+    const content =
+      `<span class="trix-attachment-spina-image" data-label="Alt text">
+       <a href="${event.detail.originalUrl}"><img src="${event.detail.embeddedUrl}" /></a>
+       </span>`
+
+    let attachment = new Trix.Attachment({content, contentType: "Spina::Image"})
```

Comment:
This just adds a link wrapper to images allowing the image to be clickable, leading to the full sized image.